NickvisionTagger.GNOME/Helpers/Help.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace NickvisionTagger.GNOME.Helpers;
+
+/// <summary>
+/// Helper class for help docs
+/// </summary>
+public static class Help
+{
+    /// <summary>
+    /// Get URL for given help page
+    /// </summary>
+    /// <param name="pageName">Help page name</param>
```

Comment:
Does this work? I think it needs to be the full shared path, but I could be wrong.

Reply:
I don't get what exactly you thought doesn't work here.

Reply:
The NickvisionTagger.GNOME.LINGUAS path...it doesn't need to be NickvisionTagger.Shared.Docs.LINGUAS?

Reply:
No, it does work like this. I actually made a typo and compiler showed me that it was trying to load `NickvisionTaggger.GNOME.LINGUAS`, so I just fixed a typo and it works.